requirements.txt

```diff
@@ -1,4 +1,9 @@
 Flask==3.0.0
 gunicorn==21.2.0
 playwright==1.40.0
-fake_useragent==1.4.0
\ No newline at end of file
+fake_useragent==1.4.0
+Pillow==9.5.0
+onnx==1.14.0
+onnxruntime==1.14.1
```

Comment:
According to the tests, onnxruntime==1.14.1 cannot be found

Reply:
They probably deleted it from PYPI for some reason. It's still on github on this link https://github.com/microsoft/onnxruntime/releases/tag/v1.14.1

I think v1.14.0 will work fine as well but we will need to test.